### DISCUSSION TITLE
建议增加SQLite数据库，这样就能很方便的在本地部署。

### DISCUSSION THREAD

**Discussion by hhun** [Qexo/Qexo#336] [Ideas]
建议增加SQLite数据库，这样就能很方便的在本地部署。
**SQLite是单文件，方便随时迁移数据。**
毕竟免费的MongoDB数据库稳定性不确定

**Comment by am-abudu** [comment]
原本就支持的

**Reply by hhun** [reply]
建议加个部署到 CloudFlare 的 Workers 的教程。

**Reply by am-abudu** [reply]
无法部署到Cloudflare Workers。如果你有方法，请告诉我

**Reply by hhun** [reply]
抱歉，忘记了Workers不支持Python

**Comment by miaowmint** [comment]
> 建议增加SQLite数据库，这样就能很方便的在本地部署。 **SQLite是单文件，方便随时迁移数据。** 毕竟免费的MongoDB数据库稳定性不确定

本来就支持的
[https://raw.githubusercontent.com/miaowmint/Qexo-docker/main/configs.py](https://raw.githubusercontent.com/miaowmint/Qexo-docker/main/configs.py)这么写配置文件就好了，稍微改一下
